repo.go

```diff
@@ -61,15 +42,7 @@ func generateHTML() {
 	input, _ := ioutil.ReadFile(readmePath)
 	body := string(
 		bluemonday.UGCPolicy().SanitizeBytes(
```

Comment:
`github_flavored_markdown.Markdown` already performs sanitization, so it's not neccessary to call `bluemonday.UGCPolicy().SanitizeBytes` on its output again.

See https://github.com/shurcooL/github_flavored_markdown/blob/3c64cb3ce00a3f4feae0c53ce7c2cc2fdacba14b/main.go#L38 and https://github.com/shurcooL/github_flavored_markdown/blob/3c64cb3ce00a3f4feae0c53ce7c2cc2fdacba14b/main.go#L90-L101.

So this can be simplified to:

```Go
body := string(gfm.Markdown(input))
```